app/cards/[id]/page.tsx

```diff
@@ -25,20 +22,14 @@ export default function CardItem() {
     }
   }, [card, router])
 
-  const [dataForChart, setDataForChart] = useState<CardRecord[]>([])
-
   function handleDelete() {
     toast.success("Карточку видалено!")
     deleteCard(id)
     router.push("/cards")
   }
 
   function createChart() {
```

Comment:
Right now that fells more like `goToChart` rather than `createChart`
```suggestion
  function goToChart() {
```